app.js

```diff
@@ -29,125 +36,128 @@ scheduleApp.config(function($stateProvider, $urlRouterProvider) {
             templateUrl: 'form-confirmation.html'
         });
 
-    $urlRouterProvider.otherwise('/index');
+    $urlRouterProvider.otherwise('/');
 })
 
-//schedule
 
-.controller('mainController', function($scope, $firebase){
+.controller('mainController', function($scope, $firebase, $state, Data){
+
+  var ref = new Firebase("https://angularschedule.firebaseio.com/days");
+  var fb = $firebase(ref);
 
- var ref = new Firebase("https://angularschedule.firebaseio.com/days");
- var fb = $firebase(ref);
+  var syncObject = fb.$asObject();
+  
+  syncObject.$bindTo($scope, 'days');
 
- var syncObject = fb.$asObject();
- 
- syncObject.$bindTo($scope, 'days');
+    $scope.selectSlot = function(slot) {
+    Data.selectedSlot = slot;
+    $state.go('form.profile');
+  }
 
- $scope.reset = function() {
+  var refs = new Firebase("https://angularschedule.firebaseio.com/booked");
 
-   fb.$set({
-     monday: {
+  $scope.selectedSlot = Data.selectedSlot;
+  $scope.addData = function(selectedSlot){
```

Comment:
Right, so make one controller for the form (which facilitates all views of the form) and one for the schedule. Both do two very different things, and don't really share any logic.